File: typescript/src/main/vsdk/toolkit/raycasting/simple/RayMatterState.ts
import { RayMatterFilterType } from "./RayMatterFilterType";

export class RayMatterState {
  public samplesPerPixel: number;
  public filter: RayMatterFilterType;

  public constructor() {
    this.samplesPerPixel = 8;
    this.filter = RayMatterFilterType.TENT_FILTER;
  }
}

File: java/base/src/main/java/vsdk/toolkit/raycasting/simple/RayMatterState.java
package vsdk.toolkit.raycasting.simple;

public class RayMatterState {
    public int samplesPerPixel; // Pixel sampling
    public RayMatterFilterType filter; // Pixel filter

    public RayMatterState() {
        samplesPerPixel = 8;
        filter = RayMatterFilterType.TENT_FILTER;
    }
}
